spec/spec_helper_account.rb

```diff
@@ -0,0 +1,52 @@
+require 'beaker-pe'
```

Comment:
Changes to acceptance setup should be done in `spec/spec_helper_acceptance.rb`.

Reply:
I was trying to add a new acceptance test that relies on automatic parameter lookup from hiera.  I was under the impression that hiera loading happens in the helper.  But since I can't get it to work, I'm yanking the test and associated helper.